deployment/playbooks/metron_install.yml

```diff
@@ -75,7 +75,8 @@
     - { role: flume,  when: install_snort | default(True) == True }
     - { role: snort , when: install_snort | default(True) == True }
     - { role: yaf, when: install_yaf | default(True) == True }
-    - { role: pcap_replay , when: pcap_replay | default(False)  == True }
+    - { role: pcap_replay , when: (pcap_replay | default(False)) or (sensor_test_mode | default(False)) == True }
+    - { role: sensor-test-mode, when: sensor_test_mode | default(False) == True }
```

Comment:
@dlyle65535 I changed this so `pcap_replay` gets installed if `sensor_test_mode` is on.  Is that the concern you had?  Any other concerns?